package.json

```diff
@@ -36,12 +36,13 @@
     "resolve": "^1.4.0"
   },
   "devDependencies": {
-    "babel-cli": "^7.0.0-beta.2",
-    "babel-core": "^7.0.0-beta.2",
+    "@babel/cli": "^7.0.0-beta.34",
+    "@babel/core": "^7.0.0-beta.34",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.34",
+    "@babel/preset-env": "^7.0.0-beta.34",
+    "babel-core": "^7.0.0-0",
```

Comment:
This is needed for jest and installs the babel-core bridge. https://github.com/babel/babel-bridge
We could have used the bridge here, but I think creating a new major version is not that big of a problem here, than it is in jest.

Reply:
Oh nice! I noticed that issue with jest when I worked on that in #76 

Thank you so much for fixing it.